### ISSUE TITLE
Option to disable auto substitution

### ISSUE THREAD

**Issue by aphadke19** [r2dev2/cryptoduel#8]
First off, thanks for sharing this tool, this is great and very helpful for kids to practice Aristocrats & Patristocrats. I have one feature request if you are willing to entertain one.

To make the problems very similar to the one kids would see on the actual Codebusters test, I wonder if you could add an option to disable the auto substitution. This happens when one makes a choice for a letter and it gets copied over to all the occurrences of that letter in the quote. While this is great for decrypting quote faster, it also enables more aggressive substitution choices, since they are relatively easy to undo.

Thoughts?

**Comment by r2dev2** [comment]
I have actually made a different website which is intended as good practice for the actual science olympiad test at https://r2dev2.github.io/garbosters/.

It mimics the online interface that scilympiad uses (including all of the features that make it hard to type in as well as their bugs). I myself used it to train for regionals and states during the years Science Olympiad was online. As of today, I am not sure if scilympiad is still used, but if it is, I recommend garbosters.

For kids to practice aristocrats and patristocrats, the only way to gain true practice for in-person codebusters is to solve the tests on paper. Solving test on paper will always be something online can't replicate because there are various strategies (eg. writing two possible substitutions, taking notes on head/tail frequency of words) that everyone has but cannot be implemented online. You can use https://toebes.com/codebusters/ along with a quote database and quickly generate hundreds of practice aristocrat/patristocrat problems (which is actually what my team did for training).

So in summary, because of above reasons and alternatives, this option is probably out of scope for cryptoduel.

Also, I am curious, which school do you coach? I wonder how far this game has spread from Lynbrook High School 😄.

**Comment by aphadke19** [comment]
Thanks, will give the new tool a try. Coaching is limited to a few kids in one of the junior teams in CUSD.
